ex03.ex

```diff
@@ -101,7 +106,15 @@ defmodule Ex03 do
 
   """
 
-  def list_equal . . . "your code"
+  def list_equal([],[]), do: true
```

Comment:
## -3

Again, try to avoid `if` and explicit comparisons if you can use pattern matching.

~~~ elixir
  def list_equal([], []),                              do: true
  def list_equal([ h | t1 ], [h | t2]),                do: list_equal(t1, t2)
  def list_equal(a, b) when is_list(a) and is_list(b), do: false
~~~